oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/MongoUtils.java

```diff
@@ -1,41 +1,32 @@
 /*
```

Comment:
I guess this class got some automatic styling from your IDE. Usually we try to avoid this kind of changes because it makes more difficult the debugging in some cases, as for example, line numbers have changed.

Would be possible to restore the unneeded changes back? Also, the license header has been modified on this and other files.

Reply:
I've just committed MongoUtils restoring original formatting, yes in the earlier stage I left my auto formatting rules in eclipse, then I removed them restoring some classes but not all, sorry I thought it was not so important